openslides/assignments/static/js/assignments/site.js

```diff
@@ -396,6 +415,22 @@ angular.module('OpenSlidesApp.assignments.site', ['OpenSlidesApp.assignments'])
         $scope.save = function(assignment) {
             Assignment.create(assignment).then(
                 function(success) {
+                    // find related agenda item
+                    Agenda.find(success.agenda_item_id).then(function(item) {
```

Comment:
I tested it with loadRelations but it does not work as expected.
```
Assignment.loadRelations(assignment, 'agenda_item').then(function() {
                        console.log(assignment.agenda_item);
```
 I have to wait until the promise is returns that I can change the loaded related agenda item. That works good with Agenda.find(). Please give me a code example if you have a better solution.